pkg/oc/cli/cmd/login/loginoptions.go

```diff
@@ -226,15 +226,10 @@ func (o *LoginOptions) gatherAuthInfo() error {
 	token, err := tokencmd.RequestToken(o.Config, o.Reader, o.Username, o.Password)
 	if err != nil {
 		suggestion := "verify you have provided the correct host and port and that the server is currently running."
+		fmt.Fprintf(o.ErrOut, "error: %v - %v\n", err, suggestion)
```

Comment:
No point in making the suggestion var now.

Reply:
thought it might help this line not get too long - will update though